docs/dev/how-to/estimate-gas.md

```diff
@@ -0,0 +1,409 @@
+# Estimate gas
+
+
+## L1 to L1 transactions
+
+To estimate gas for L1 to L1 transactions, use the `eth_estimateGas` method.
```

Comment:
the important bit -- this eth_estimateGas should be sent do some L1 server - not to zksync, right?

Reply:
correct

Reply:
will deal with this